packages/cbioportal-clinical-timeline/src/TimelineTrack.tsx

```diff
@@ -48,26 +103,74 @@ function renderSuperscript(number: number) {
     );
 }
 
+function renderLineChartLine(points: { x: number; y: number }[]) {
```

Comment:
seems like it would be nice to have this outside of TimelineTrack.   Or some subtype of TimelineTrack.  I mean, suppose we have other types